### PR TITLE
MINOR: doc: fix typo in config-streams

### PR DESCRIPTION
**Pull request by iblislin**

### CODE REVIEW

**Review by chia7712 — APPROVED** [review]
LGTM

File: docs/streams/developer-guide/config-streams.html [hunk 1]
```diff
@@ -216,7 +216,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
 //streamsSettings.put(StreamsConfig.REPLICATION_FACTOR_CONFIG, 3);
 streamsSettings.put(StreamsConfig.topicPrefix(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG), 2);
 streamsSettings.put(StreamsConfig.producerPrefix(ProducerConfig.ACKS_CONFIG), "all");
```
Comment (chia7712) [inline docs/streams/developer-guide/config-streams.html:218]: Could you please fix line#217 and line#218 also?

File: docs/streams/developer-guide/config-streams.html [hunk 2]
```diff
@@ -216,7 +216,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
 //streamsSettings.put(StreamsConfig.REPLICATION_FACTOR_CONFIG, 3);
 streamsSettings.put(StreamsConfig.topicPrefix(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG), 2);
```
Comment (iblislin) [inline docs/streams/developer-guide/config-streams.html:217]:
```suggestion
streamsSettings.put(StreamsConfig.topicPrefix(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG, 2);
```

File: docs/streams/developer-guide/config-streams.html (hunk 1 above)
Comment (iblislin) [reply docs/streams/developer-guide/config-streams.html:218]: well, I think line#217 and line#218 are legit

**Comment by chia7712** [comment]
@iblislin thanks for patch!

**Pull request merged**